static/js/apps/eval_rig/feedback_form.tsx

```diff
@@ -84,103 +92,84 @@ export function FeedbackForm(props: FeedbackFormProps): JSX.Element {
-          />
-        </div>
+          <div>
+            <h2>GEMMA MODEL EVALUATION</h2>
+            <h3>{props.evalInfo.question}</h3>
+            <h3>{props.evalInfo.llmResponse}</h3>
+            <OneQuestion
+              question="Question from the model"
+              name="question"
+              options={{
+                DC_QUESTION_IRRELEVANT: "Irrelevant, vague, requires editing",
+                DC_QUESTION_RELEVANT: "Well formulated & relevant",
+              }}
+              handleChange={handleChange}
+              responseField={response.question}
+              disabled={completed}
+            />
+            <OneQuestion
+              question="Model response quality"
+              name="llmResponse"
+              options={{
+                LLM_STAT_ACCURATE: "Stats seem accurate",
+                LLM_STAT_INACCURATE: "Stats seem inaccurate",
+              }}
+              handleChange={handleChange}
+              responseField={response.llmResponse}
+              disabled={completed}
+            />
+          </div>
 
-        <div>
-          <h2>DATA COMMONS EVALUATION</h2>
-          <h3>{props.evalInfo.dcResponse}</h3>
-          <h3>{props.evalInfo.dcStat}</h3>
-          <OneQuestion
-            question="Response from Data Commons"
-            name="dcResponse"
-            options={dcResponseOptions}
-            handleChange={handleChange}
-            responseField={response.dcResponse}
-          />
+          <div>
+            <h2>DATA COMMONS EVALUATION</h2>
+            <h3>{props.evalInfo.dcResponse}</h3>
+            <h3>{props.evalInfo.dcStat}</h3>
+            <OneQuestion
+              question="Response from Data Commons"
+              name="dcResponse"
+              options={dcResponseOptions}
+              handleChange={handleChange}
+              responseField={response.dcResponse}
+              disabled={completed}
+            />
 
-          <OneQuestion
-            question="Response from Data Commons"
-            name="dcStat"
-            options={{
-              DC_STAT_ACCURATE: "Stats seem accurate",
-              DC_STAT_INACCURATE: "Stats seem inaccurate",
-            }}
-            handleChange={handleChange}
-            responseField={response.dcStat}
-          />
-        </div>
+            <OneQuestion
+              question="Response from Data Commons"
+              name="dcStat"
+              options={{
+                DC_STAT_ACCURATE: "Stats seem accurate",
+                DC_STAT_INACCURATE: "Stats seem inaccurate",
+              }}
+              handleChange={handleChange}
+              responseField={response.dcStat}
+              disabled={completed}
+            />
+          </div>
 
-        <button type="submit" disabled={Object.values(response).includes("")}>
-          Submit
-        </button>
-      </fieldset>
-    </form>
+          <button type="submit" disabled={Object.values(response).includes("")}>
```

Comment:
nit: maybe only include submit button if not `completed`?

Reply:
good idea!